example/botpress/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function(bp) {
+  bp.middlewares.load();
+
+  //Catch 'hello world' from 'facebook'
+  bp.hear({
+    platform: 'slack',
+    type: 'message',
+    text: 'hello world'
+  }, (event, next) => {
+    console.log("HERE RECEUVED");
```

Comment:
Somewhere here I want to see **reply** function call (or something like that), which will pass data back to broid-kit